index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Minesweeper</title>
+    <link rel="stylesheet" href="style.css" />
+    <script type="module" src="script.js"></script>
+</head>
+
+<body>
+    <header>
+        <h1>Minesweeper</h1>
+    </header>
+    <div></div>
+    <main>
+        <article>
+            <div class="game-options">
+                <button class="new-game-btn" title="New Game">
+                    <img src="restartIcon.svg" class="icon-btn" />
+                </button>
+                <button class="flag-btn" title="turn flagging on">
+                    <img src="flagIconWhite.svg" class ="icon-btn"/>
+                </button>
+            </div>
+            <ul class="container"></ul>
```

Comment:
container isn't the most descriptive name, is it a
```suggestion
            <ul class="grid-container"></ul>
```
??